sparse_strips/vello_common/src/coarse.rs

```diff
@@ -9,7 +9,7 @@ use vello_api::paint::Paint;
 use vello_api::peniko::Fill;
 
 /// The width of a wide tile.
-pub const WIDE_TILE_WIDTH: usize = 256;
+pub const WIDE_TILE_WIDTH: usize = WideTile::WIDTH as usize;
```

Comment:
These changes are unfortunate, but I don't think doing the clean up in this PR is the right timing. When left-of-viewport tile culling lands and the x-coordinates go unsigned, is probably a better time.